README.md

```diff
@@ -2,34 +2,138 @@
 These authorized users can then submit tournament brackets to populate rating data for the season.
+Batch addition of tournaments (for pools) is supported.
+Users can submit indvidiual matches for approval to play for ELO. Authorized users can approve these matches.
 
 The match history for these games are then stored locally in a DB.
 
-Players can be given an initial seed from a previous season or seeded directly by an authorized official.
+Players can be given an initial seed from a previous season,  by an authorized official or by a default value.
 
-Users can visualize the match data with different types of sorting and inspect indivisual players.
+Users can view match data with different types of sorting and inspect indivisual players.
 Player match data can also be filtered to show victories/losses and upsets.
+Users can get a list of matches played between two players.
+
+Users can see the top N players in the season or all the players in the season.
 
-The season official can also create divisions within the leaderboard by score. These divisions can be used by tournament officials to incentivise competition. Additionally divisions within a leaderboard can be used for round robin side events or general seeding.
+The season official can also create divisions within the leaderboard by score.
+These divisions can be used by tournament officials to incentivise competition.
+Additionally divisions within a leaderboard can be used to produce round robin side events or general seeding.
 
 At the end of the season it can be marked as finished and a subjective power rankings can be submitted.
+Users can see how many days left 
+
 Panelists involved in the subjective power rankings can utilize the upset filtering to support their decisions.
 
 #Resources
 
-Seasons
-Tournaments
-Matches
-Players
+- User
 
 #Response Codes
-200 Ok
-400 Invalid Challonge Reference, could not validate.
-401 User isn't authorized to submit tournament.
-404 Resource does not exist.
-409 Duplicate tournament submission.
-500 Internal Error.
+
+- 200 Ok
+- 400 Invalid Challonge Reference, could not validate.
+- 401 User isn't authorized to submit tournament.
+- 404 Resource does not exist.
+- 409 Duplicate tournament submission.
+- 500 Internal Error.
+
+#Swagger
+See RestPRApi.yaml for Swagger specification.
+
+#Configure
+Define a database connection in configuration.yaml using the database: key.
+
+You'll need to use ojdbc6_g.jar from Oracle in your /bin folder.
+
+#Build
+Run the build gradle task:
+	 gradle build
+
+#Run
+To run the application you must execute this command:
+	 java -classpath bin/ojdbc6_g.jar:build/libs/RESTPR-all.jar edu.oregonstate.mist.restpr.RESTPRApplication server configuration.yaml
+
+or just run manualjavacall.sh:
+	 ./manualjavacall.sh
+
+#Connection Examples
+The following examples can be excuted with netcat using heredocs like this:
+	nc localhost 8008 << HERE
```

Comment:
i'm not sure why github is formatting some of your code blocks and not others, but you can explicitly define code blocks in markdown by indenting 4 spaces or by surrounding code with triple backticks